code/game/bioscans.dm

```diff
@@ -70,6 +72,8 @@ GLOBAL_DATUM_INIT(bioscan_data, /datum/bioscan_data, new)
 	for(var/mob/living/carbon/human/current_human as anything in GLOB.alive_human_list)
 		if(current_human.mob_flags & NOBIOSCAN)
 			continue
+		if(istype(get_turf(current_human), /area/adminlevel/ert_station/fax_response_station))
+			continue
```

Comment:
these should be flags or traits on the area that make them completely immune to bioscans, not snowflake type checks

Reply:
imported forest code